src/Bridges/SymfonyConsole/ResetCommand.php

```diff
@@ -16,12 +16,9 @@
 
 class ResetCommand extends BaseCommand
 {
-	/** @var string */
-	protected static $defaultName = 'migrations:reset';
-
```

Comment:
Default name should not be removed, even if not used in configure() because it breaks lazy loading.

Also commands with custom name can support lazy loading. In case of contributte/console could be added service tag `$service->addTag('console.command', 'custom:command:name')`. Not sure about kdyby/console and symfony DI integrations.

Reply:
Actually I wanted to not remove this static property but symfony-bundle test failed on a weird notice like "undefined index **continue**" in Symfony/Application.php. 

I didn't know this is symfony "build-in feature" from `v3.4.10` and I have the same opinion like https://twitter.com/Majkl578/status/1214523751603286016

Any ideas how to resolve it? I agree that breaking lazy loading is not good.